state/leaseupgrades_test.go

```diff
@@ -15,8 +15,6 @@ import (
 	coretesting "github.com/juju/juju/testing"
 )
 
-var _ = gc.Suite(&upgradesSuite{})
```

Comment:
Was this registered elsewhere?

Reply:
Yes, this was registered in upgrades_test.go.  The fact that it was registered here too meant that we were running all the tests twice.